modal/runner/containers/vllm_unified.py

```diff
@@ -128,14 +130,17 @@ def __init__(self):
     gpu=modal.gpu.A10G(count=1),
     concurrent_inputs=4,
     max_containers=5,
+    quantization="GPTQ",
 )
 
-# TODO: quantize this one too. shipping the others first to limit blast radius
+_psyfighter2 = "TheBloke/LLaMA2-13B-Psyfighter2-GPTQ"
 VllmContainer_KoboldAIPsyfighter2 = _make_container(
     name="VllmContainer_KoboldAIPsyfighter2",
-    model_name="KoboldAI/LLaMA2-13B-Psyfighter2",
-    gpu=modal.gpu.A100(count=1, memory=40),
-    concurrent_inputs=32,
+    model_name=_psyfighter2,
+    gpu=modal.gpu.A10G(count=1),
+    concurrent_inputs=4,
+    max_containers=5,
+    quantization="GPTQ",
```

Comment:
we get like max 10 requests a minute for this model.  think this allocation should be more than enough, might be able to further reduce batch size for better throughput floor

will try this first tho